setup.cfg

```diff
@@ -46,6 +46,7 @@ pipx.run =
 
 [options.extras_require]
 docs =
+    build[typing]
```

Comment:
Shouldn't we do here a self install? A reference to ourselves feels odd :thinking:

Reply:
It's supposed to be a supported/encouraged idiom (though it didn't work properly before pip 21.2).

Reply:
https://github.com/pypa/pip/issues/10393#issuecomment-942771191

Reply:
So from where is this build pulled? PyPi or the local source tree?

Reply:
So this is a way to make some extras depend on other extras :thinking: within the package?

Reply:
> So from where is this build pulled? PyPi or the local source tree?

From the local source tree when installing locally (with `.`).



> So this is a way to make some extras depend on other extras 🤔 within the package?

Indeed.